features/restaurant_grouping.feature

```diff
@@ -9,11 +9,11 @@ Feature: User can select restaurant by category
       | Thai    | Thai food    |
       | Mexican | Mexican food |
     And the following restaurants exist
-      | name       | address     | restaurant_category |
-      | My Thai    | Stockholm   | Thai                |
-      | My Mexican | Mexico city | Mexican             |
+      | name       | address     | restaurant_category |  description              |
+      | My Thai    | Stockholm   | Thai                |  Some hip thai place      |
+      | My Mexican | Mexico city | Mexican             |  Some nice mexican place  |
```

Comment:
Hmm I'm just wondering if we should stick to one city across the project because we haven't started on implementing the delivery feature

Reply:
I don't think that's relevant to the delivery feature. Plus the user could as well be in Mexico. :)